### PR TITLE
docs: Guide to running a beta-5 node

### PR DESCRIPTION
**Pull request by calldelegation**
- [ ] get client team to review
- [x] address all the TODOs in running a beta-5 node
- [ ] inquire about how installations work

### CODE REVIEW

File: docs/guides/docs/running-a-node/running-a-beta-5-node.mdx [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+fuel-core run \
+--service-name {ANY_SERVICE_NAME} \
+--keypair {P2P_SECRET} \
+--relayer {ETH_RPC_ENDPOINT}\
+--ip 0.0.0.0 --port 4000 --peering-port 30333 \
+--db-path  ~/.fuel_beta5 \
+--chain ./chainConfig.json \
+--utxo-validation --poa-instant false --enable-p2p \
+--min-gas-price 1 --max-block-size 18874368  --max-transmit-size 18874368 \
+--bootstrap-nodes /dns4/p2p-beta-4.fuel.network/tcp/30333/p2p/16Uiu2HAm3xjsqASZ68KpaJPkPCMUiMgquhjyDHtxcVxVdFkMgRFf,/dns4/p2p-beta-4.fuel.network/tcp/30334/p2p/16Uiu2HAmJyoJ2HrtPRdBALMT8fs5Q25xVj57gZj5s6G6dzbHypoS \
```
Comment (xgreenx) [inline docs/guides/docs/running-a-node/running-a-beta-5-node.mdx:166]:
```suggestion
--reserved-nodes /dns4/p2p-beta-4.fuel.network/tcp/30333/p2p/16Uiu2HAm3xjsqASZ68KpaJPkPCMUiMgquhjyDHtxcVxVdFkMgRFf,/dns4/p2p-beta-4.fuel.network/tcp/30334/p2p/16Uiu2HAmJyoJ2HrtPRdBALMT8fs5Q25xVj57gZj5s6G6dzbHypoS \
```

File: docs/guides/docs/running-a-node/running-a-beta-5-node.mdx [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+  file="./running-a-beta-4-node.mdx"
+  comment="generate_fuel_p2p_key_end"
+  commentType="{/*"
+/>
+
+## Chain Configuration
+
+{/*TODO: Update link for beta-5*/}
+
+To run a local node with persistence, you must configure a `chainConfig.json` file. Here is the [specific configuration](https://github.com/FuelLabs/fuel-core/blob/v0.20.4/deployment/scripts/chainspec/beta_chainspec.json) for `beta-5`:
```
Comment (xgreenx) [inline docs/guides/docs/running-a-node/running-a-beta-5-node.mdx:139]: https://github.com/FuelLabs/fuel-core/blob/v0.22.0/deployment/scripts/chainspec/beta_chainspec.json

File: docs/guides/docs/running-a-node/running-a-beta-5-node.mdx (hunk 1 above)
Comment (xgreenx) [reply docs/guides/docs/running-a-node/running-a-beta-5-node.mdx:166]: Also we need to get correct IPs for beta5 network. @tirkesi Could you post here or in the slack what DNS are correct?=)

File: docs/guides/docs/running-a-node/running-a-beta-5-node.mdx [hunk 3]
```diff
@@ -0,0 +1,202 @@
...
+--ip 0.0.0.0 --port 4000 --peering-port 30333 \
+--db-path  ~/.fuel_beta5 \
+--chain ./chainConfig.json \
+--utxo-validation --poa-instant false --enable-p2p \
+--min-gas-price 1 --max-block-size 18874368  --max-transmit-size 18874368 \
+--bootstrap-nodes /dns4/p2p-beta-4.fuel.network/tcp/30333/p2p/16Uiu2HAm3xjsqASZ68KpaJPkPCMUiMgquhjyDHtxcVxVdFkMgRFf,/dns4/p2p-beta-4.fuel.network/tcp/30334/p2p/16Uiu2HAmJyoJ2HrtPRdBALMT8fs5Q25xVj57gZj5s6G6dzbHypoS \
+--sync-header-batch-size 1000 \
+--enable-relayer \
+--relayer-v2-listening-contracts 0xE6B0E27F85abaCfC5149642c30F4BE9a878Aa4e9 \
+--relayer-da-deploy-height 8538323 \
```
Comment (xgreenx) [inline docs/guides/docs/running-a-node/running-a-beta-5-node.mdx:170]: The same here, we need actual values @tirkesi

File: docs/guides/docs/running-a-node/running-a-beta-5-node.mdx (hunk 1 above)
Comment (tirkesi) [reply docs/guides/docs/running-a-node/running-a-beta-5-node.mdx]: sentry-0 PUBLIC_ADDRESS:  /dns4/p2p-beta-5.fuel.network/tcp/30333/p2p/16Uiu2HAmSMqLSibvGCvg8EFLrpnmrXw1GZ2ADX3U2c9ttQSvFtZX

sentry-1 PUBLIC_ADDRESS: /dns4/p2p-beta-5.fuel.network/tcp/30334/p2p/16Uiu2HAmVUHZ3Yimoh4fBbFqAb3AC4QR1cyo8bUF4qyi8eiUjpVP

---

endpoint that handles the tcp connection: k8s-beta5-beta5tcp-abf39034d2-db60da65b4fcac49.elb.us-east-1.amazonaws.com

```
nc -z -v k8s-beta5-beta5tcp-abf39034d2-db60da65b4fcac49.elb.us-east-1.amazonaws.com 30333                                    
Connection to k8s-beta5-beta5tcp-abf39034d2-db60da65b4fcac49.elb.us-east-1.amazonaws.com port 30333 [tcp/*] succeeded!
```

File: docs/guides/docs/running-a-node/running-a-beta-5-node.mdx (hunk 3 above)
Comment (tirkesi) [reply docs/guides/docs/running-a-node/running-a-beta-5-node.mdx:170]:
```
      relayerV2ListeningContracts:
        - 0x557c5cE22F877d975C2cB13D0a961a182d740fD5 # FuelMessagePortal
      relayerDADeployHeight: 4867877
```

**Review by sarahschwartz — APPROVED** [review]

**Pull request merged**